libafl/src/executors/timeout.rs

```diff
@@ -277,13 +319,54 @@ where
 
             write_volatile(&mut data.timeout_input_ptr, core::ptr::null_mut());
 
-            self.windows_reset_timeout()?;
```

Comment:
previously I made two functions for windows timeouts
`windows_reset_timeout()` and `windows_delete_timer_queue()`.
(They are pretty confusing, sorry about that)
They two do different things. 
`reset_timeout()` is about setting the timer to 0 sec
`delete_timer_queue()` is about wiping the timer object off.

So in post_run_reset we should do the same as reset_timeout() because we just want to reset it to zero

but in crash_handler we should do what delete_timer_queue() does because we want to wipe the entire timer object